db/migrate/20220520185715_create_users.rb

```diff
@@ -0,0 +1,15 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :firsName, null: false
```

Comment:
A modo de convención de en Ruby on Rails, los nombres compuestos deben ir escritos en snake case:
```suggestion
      t.string :firsName, null: false
```
```suggestion
      t.string :first_name, null: false
```